gateway/ln-gateway/src/rpc/mod.rs

```diff
@@ -52,7 +52,7 @@ pub struct DepositAddressPayload {
     pub federation_id: FederationId,
 }
 
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[serde(rename_all = "snake_case")]
```

Comment:
Can't deserialize `all` as `BitcoinAmountOrAll`

Reply:
It must be on `BitcoinAmountOrAll`